.gitignore

```diff
@@ -5,6 +5,7 @@
 **/liferay/logs/*.log
 **/liferay/logs/*.xml
 **/pmd-reports
+**/TEST-frontend-js.xml

```

Comment:
I don't think We can touch gitignore files

Reply:
This `.gitignore` rule was added in order to avoid some mistake from a developer, cause this file is auto-generated whenever a `yarn format` is performed.  It's is isolated into the commit 47e79abd9818c90ce428d260ae9e26acbbc7ad2a, so it can be easily revert if any issues occurs.